src/components/graph-view.js

```diff
@@ -184,6 +184,14 @@ class GraphView extends React.Component<IGraphViewProps, IGraphViewState> {
   }
 
   componentDidMount() {
+    if (!this.state.mounted) {
+      this.setState({
+        mounted: true,
+      });
+    }
+  }
```

Comment:
i think this is considered a bad practice as it will potentially cause an extra render?

Reply:
I added this to ensure that the layout run in componentDidUpdate gets called as soon the component renders. It's technically allowed [per the docs](https://reactjs.org/docs/react-component.html#componentdidmount) though in this case I think we could simply invoke the layout at componentDidMount in addition to componentDidUpdate and avoid the additional rerender, I'll update.